package.json

```diff
@@ -292,9 +292,11 @@
 	},
 	"resolutions": {
 		"react-dates": "17.1.1",
-		"@automattic/newspack-blocks/swiper@4.5.1": "patch:swiper@4.5.1#.patches/swiper@4.5.1.diff",
-		"@automattic/newspack-blocks/prettier": "npm:wp-prettier@2.6.2-beta-1",
 		"@types/react": "^17.0.39",
+		"newspack-components/@wordpress/base-styles": "4.5.0",
+		"newspack-components/@wordpress/components": "19.15.0",
+		"newspack-components/@wordpress/element": "4.7.0",
+		"newspack-components/@wordpress/i18n": "4.9.0",
 		"keytar@npm:7.7.0/node-addon-api": "3.1.0",
```

Comment:
This is an interesting change.

In wp-calypso, we currently use `@wordpress/components` v19.15.0 (which is what's resolved in the yarn.lock file).

newspack-components (a dependency of newspack-blocks) uses `@wordpress/components` v19.0.1. Instead of using our _existing_ resolution -- 19.15.0 -- yarn instead decides to get a more recent version of `@wordpress/components` which depends on other newer versions of WordPress packages. At least one of these WordPress packages isn't considered a breaking change, so it gets updated to a _very_ recent version of WordPress components.

Unfortunately, `@wordpress/*` packages don't work very well when versions are somewhat misaligned. In this case, we end up pulling in the react 18 dependency, which also ends up completely upgrading some `@type/` packages with poorly specified version constraints (e.g. >= 16.0), and we end up with a fair number of dependencies changing in the tree.

All of which means, without this, the dependency upgrades would be incompatible with Calypso.

Reply:
Note: I want to double test the assumption that this broke things once we get everything working.

Reply:
It sounds like upgrading `@wordpress/components` in Newspack Components to v19.15.0 would solve this in the future?